packages/lodestar/src/sync/gossip/handler.ts

```diff
@@ -49,7 +50,9 @@ export class BeaconGossipHandler implements IGossipHandler {
     this.subscribe(forkDigest);
   };
 
-  private subscribe = (forkDigest: ForkDigest): void => {
+  private subscribe = async (forkDigest: ForkDigest): Promise<void> => {
+    // TODO: remove this once we fix in gossipsub-js
```

Comment:
@wemeetagain I was not able to subscribe to `/beacon_block/` immediately after we start gossip (`/beacon_block/` was not even shown in `gossip subscriptions` log) so I have to delay 100ms like this in order to pass e2e.